as/cNFT/assembly/models/persistent_tokens.ts

```diff
@@ -113,6 +113,50 @@ export class PersistentTokens {
         return token
     }
 
+    /**
+     * @param tokenId: Id of the token
+     * @param bidderId: accountId of the bidder
+    */
+
+    cnft_transfer(tokenId: TokenId, bidderId: AccountId): void {
+
+        /* Getting stored token from tokenId */
+
+        const token = this._tmap.getSome(tokenId)
```

Comment:
I think it's better to get the token using `this.get(tokenId)`. So in future, we have to change it in just one place

Reply:
Yeah. agreed :-)